docs/how_to/hypervisors/vagrant_libvirt.md

```diff
@@ -0,0 +1,54 @@
+# Vagrant Libvirt #
+
+This driver enables to trigger tests using libvirtd daemon. It is
+based
+on [libvirt](https://github.com/vagrant-libvirt/vagrant-libvirt)'s
+plugin for vagrant.
+
+## Basic Options ##
+
+With the libvirt daemon on the same host than the running test command
+a simple configuration is
+
+```yaml
+HOSTS
+  centos-puppet-keystone:
+    hostname: puppet-keystone.example.net
+    roles:
+      - master
+    platform: el-7-x86_64
+    box: centos/7
+    box_url:  http://cloud.centos.org/centos/7/vagrant/x86_64/images/CentOS-7.LibVirt.box
+    hypervisor: vagrant_libvirt
+    vagrant_memsize: 4096
+    vagrant_cpus: 2
+```
+
+Those are the usual beaker parameters. Note the `hypervisor`
+parameter. Multiple vms is supported.
+
+
+## Advanced Options and remote libvirt daemon ##
+
+This driver gives you all available parameters from the
+vagrant-libvirt plugin. Beware there could be dragon here as beaker
+has some expectation about the created vms.
+
+To pass them down you add the them in the config section, here is a
+example.
+
+```yaml
+CONFIG:
+  libvirt:
+    uri: qemu+ssh://root@libvirt.system.com/system
+```
+
+This is the one of the most useful one. It enables one to connect to a
+remote libvirt daemon to do the testing. You'll have to make sur the
+network is working yourself. `management_network_name` and
+`management_network_address` can be useful here as well.
+
+Another good cadidate is `volume_cache: unsafe`.
+
+Check out the documentation in the vagrant plugin repository for a
```

Comment:
would you please link to the vagrant plugin repository here so we can just click through to that?